tests/lib.rs

```diff
@@ -29,13 +29,9 @@
 #![allow(box_pointers, fat_ptr_transmutes, missing_copy_implementations,
          missing_debug_implementations, variant_size_differences)]
 
-#![cfg_attr(feature="clippy", feature(plugin))]
-#![cfg_attr(feature="clippy", plugin(clippy))]
-#![cfg_attr(feature="clippy", deny(clippy))]
-
 #[macro_use]
-#[cfg_attr(feature="clippy", allow(useless_attribute))]
-#[allow(unused_extern_crates)]  // Only using macros from maidsafe_utilites
+#[cfg_attr(feature="cargo-clippy", allow(useless_attribute))]
+#[allow(unused_extern_crates)] // Only using macros from maidsafe_utilites
```

Comment:
is this still required (also what macros are we using from utilities ?) Check other places asnd pr's too